File: python/src/core/models/entities/agent_log_entry.py
from datetime import datetime
from typing import Optional
from pydantic import Field
from core.models.entities.abstract.base_entity import BaseEntity


class AgentLogEntry(BaseEntity):
    agent_id: str = Field(alias="agent_id")
    log_text: Optional[str] = Field(default=None, alias="log_text")
    created_date: Optional[datetime] = Field(
        default=None, alias="created_date")

    class Config:
        exclude_none = True
        populate_by_name = True

File: core/sdk/dotnet/Models/Entities/AgentLogEntry.cs
using Agience.Core.Models.Entities.Abstract;
using System.Text.Json.Serialization;

namespace Agience.Core.Models.Entities
{
    public class AgentLogEntry : BaseEntity
    {
        [JsonPropertyName("agent_id")]
        public string AgentId { get; set; } = null!;

        [JsonPropertyName("log_text")]
        public string? LogText { get; set; }

        [JsonPropertyName("created_date")]
        public DateTime? CreatedDate { get; set; }
    }
}